python/tflite_micro/signal/ops/fft_ops_test.py

```diff
@@ -33,6 +33,31 @@ def GetResource(self, filepath):
       file_text = f.read()
     return file_text
 
+  def SingleFftAutoScaleTest(self, filename):
+    lines = self.GetResource(filename).splitlines()
+    func = tf.function(fft_ops.fft_auto_scale)
+    input_size = len(lines[0].split())
+    concrete_function = func.get_concrete_function(
+        tf.TensorSpec(input_size, dtype=tf.int16))
+    interpreter = util.get_tflm_interpreter(concrete_function, func)
+    i = 0
+    while i < len(lines):
+      in_frame = np.array([int(j) for j in lines[i].split()], dtype=np.int16)
+      out_frame_exp = [int(j) for j in lines[i + 1].split()]
+      scale_exp = [int(j) for j in lines[i + 2].split()]
+      # TFLite
```

Comment:
TFLM?

Reply:
Done. Changed to TFLM, but we have a TODO elsewhere to revisit these tests in general when we get the chance.